documentation/modules/operators/ref-operator-cluster.adoc

```diff
@@ -171,6 +171,51 @@ If you are using a different DNS domain name suffix in your cluster, change the
 The timeout for building new Kafka Connect images with additional connectots, in milliseconds.
 This value should be increased when using Strimzi to build container images containing many connectors or using a slow container registry.
 
+`STRIMZI_FEATURE_GATES`:: Optional.
+Disables or enables feature gates to activate new features or change the operator behavior.
+For more details about the feature gates, see xref:ref-operator-cluster-feature-gates-{context}[].
+
+[id='ref-operator-cluster-feature-gates-{context}']
+== Feature Gates
+
+Strimzi operators supports feature gates to gate selected functionality and switch it on or off.
+Each feature gate has a default state (enabled or disabled by default) and users are to manually enable or disable them.
+The feature gates always apply for the whole operator and alter its behavior.
+
+The feature gates should go through 3 different maturity stages:
+* Alpha (typically disabled by default)
+* Beta (typically enabled by default)
+* General Availability / GA (typically enabled by default or always enabled)
+
+Alpha stage indicates that the feature might be unstable (it might change in the future), experimental or not sufficiently tested for production use.
+Beta stage indicates a that the feature is already well tested and its functionality should most probably not change.
+GA means that the feature is stable and should not change anymore.
+GA features might optionally also change to always enabled features.
+In this case the feature gate will be removed, and it will not be possible to disable the feature anymore.
+The alpha and beta features might be removed in case they are deemed as not useful or not the solution.
```

Comment:
```suggestion
Alpha stage features might be unstable, subject to change, experimental, or not sufficiently tested for production use.
Beta stage features are already well tested and their functionality is not likely to change.
GA means that the feature is stable and should not change.
GA feature gates might also change to features that are always enabled.
In this case, the feature gate is removed and it will not be possible to disable the feature.
Alpha and beta features are removed if they do not prove to be useful.
```